### PR TITLE
rgw/doc: fix indent

### PR DESCRIPTION
**Pull request by Yan-waller**
paragraphs(or new line) with a blank line:

doc

Signed-off-by: Yan Jun <yan.jun8@zte.com.cn>

### CODE REVIEW

**Comment by xiexingguo** [comment]
:+1:

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -48,7 +51,7 @@ mirroring data changes between the Ceph Object Gateway instances and
 this allows for a much simpler configuration scheme and active-active
 configurations. Please note that metadata operations such as creating
 a user would still need to go through the master zone, however data
-operations such as creation of buckets objects can be handled by any
+operations such as creation of buckets or objects can be handled by any
```
Comment (cbodley) [inline doc/radosgw/multisite.rst]: this isn't accurate; bucket creation is a metadata operation

Comment (theanalyst) [inline doc/radosgw/multisite.rst]: hmm actually we need to mention that bucket creation, if done on a non-master zone,  internally forwards the request to the master region (this sort of didn't apply to the user creation as we aren't exactly going via the rest api and thus directly create users on a non-master zone)

File: doc/radosgw/multisite.rst [hunk 2]
```diff
@@ -31,10 +31,13 @@ non-master zones. Following are the basic terminologies that would be used:
   is incremented on every commit operation. Every realm has an associated
   current period, holding the current state of configuration of the zonegroups
   and storage policies. Any configuration change for a non-master zone will
-  increment the period's epoch. Changing the master zone to a different zone
+  increase the period's epoch. Changing the master zone to a different zone
```
Comment (theanalyst) [inline doc/radosgw/multisite.rst]: we use increment everywhere else, though I'm not really pedantic about that, and okay if this makes it or not :)

Comment (Yan-waller) [inline doc/radosgw/multisite.rst]: hmm, maybe you are right. but from my point a verb `increase ` sounds reasonable here :)

Comment (cbodley) [inline doc/radosgw/multisite.rst]: i think i prefer increment, because it's more specific in that it increases by 1

Comment (Yan-waller) [inline doc/radosgw/multisite.rst]: oh, I got it. @theanalyst @cbodley thanks you two. dropped it

**Comment by Yan-waller** [comment]
I retain the indent modification since that looks more clearer.

**Pull request merged**